contracts/exchange/api/RiskCheck.sol

```diff
@@ -2,15 +2,45 @@
 pragma solidity ^0.8.20;
 
 import "./BaseContract.sol";
+import "./ConfigContract.sol";
 import "../types/DataStructure.sol";
 import "../util/Asset.sol";
 import "../util/BIMath.sol";
 
-contract RiskCheck is BaseContract {
+struct MaintenanceMarginConfig {
+  BI size;
+  BI ratio;
+}
+
+// The maximum number of maintenance margin tiers
+uint256 constant MAX_M_MARGIN_TIERS = 12;
+// The bit mask for the least significant 32 bits
+uint256 constant LSB_32_MASK = 0xFFFFFFFF;
+// The bit mask for the least significant 24 bits, used for Kind, Underlying, Quote encoding in determining the insurance fund subaccount ID
+bytes32 constant KUQ_MASK = bytes32(uint256(0xFFFFFF));
+
+contract RiskCheck is BaseContract, ConfigContract {
```

Comment:
Although this is logical to move the liquidation into this block, my restraint from refactoring is that this will produce a large diff and cause confusion for Spearbit. 

How about we keep it as is for now, and once we address all the issues, we can refactor in one go?

Reply:
I am OK with this. Updated.

Reply:
Will remove liquidation contract altogether.

Reply:
Done.